scheduler/src/cook/mesos/task.clj

```diff
@@ -81,7 +81,11 @@
         custom-executor? (use-custom-executor? job-ent)
         cook-executor? (and (not container) ;;TODO support cook-executor in containers
                             (use-cook-executor? job-ent executor-config))
+        job-uuid (:job/uuid job-ent)
+        group-uuid (util/job-ent->group-uuid job-ent)
         environment (cond-> (util/job-ent->env job-ent)
+                            group-uuid (assoc "COOK_JOB_GROUP_ID" (str group-uuid))
+                            job-uuid (assoc "COOK_JOB_ID" (str job-uuid))
```

Comment:
When would `job-uuid` be falsy?

Reply:
Should these be `_UUID` instead of `_ID`?

Reply:
I like `_ID` as Mesos names its variables `MESOS_EXECUTOR_ID` and `MESOS_SLAVE_ID`.

Reply:
Sure, but in Cook we already refer to job uuid and group uuid in other places

Reply:
> When would job-uuid be falsy?

Avoided the condition check.

Reply:
Added the `_UUID` suffix to the variable names.